core/controlplane/config/config.go

```diff
@@ -488,9 +493,10 @@ type DeploymentSettings struct {
 	DeprecatedInternetGatewayID           string                `yaml:"internetGatewayId,omitempty"`
 	InternetGateway                       model.InternetGateway `yaml:"internetGateway,omitempty"`
 	// Required for validations like e.g. if instance cidr is contained in vpc cidr
-	VPCCIDR                   string            `yaml:"vpcCIDR,omitempty"`
-	InstanceCIDR              string            `yaml:"instanceCIDR,omitempty"`
-	K8sVer                    string            `yaml:"kubernetesVersion,omitempty"`
+	VPCCIDR                   string `yaml:"vpcCIDR,omitempty"`
+	InstanceCIDR              string `yaml:"instanceCIDR,omitempty"`
+	K8sVer                    string `yaml:"kubernetesVersion,omitempty"`
+	KubeAWSVersion            string
```

Comment:
Note to self: I'll probably move this somewhere else, because the `DeploymentSettings` is a container that reflects a the part of `cluster.yaml`. Assuming we won't want to allow setting `KubeAWsVersion`  via cluster.yaml, it would be natural to move it.

Not sure where to move yet.